linux_os/guide/services/obsolete/inetd_and_xinetd/package_tcp_wrappers_installed/rule.yml

```diff
@@ -17,12 +17,14 @@ rationale: |-
 severity: medium
 
 identifiers:
+    cce@rhel6: 82430-0
     cce@rhel7: 27361-5
+    cce@rhel8: 82429-2
```

Comment:
@shawndwells please return the `rhel8` CCE back to the pool. `tcp_wrappers` is not a package in that release.